cf/test/test_Field.py

```diff
@@ -460,28 +460,22 @@ def test_Field_collapse(self):
                     b = getattr(f.data, method)(axes=axes)
                     self.assertTrue(
                         a.equals(b, rtol=1e-05, atol=1e-08, verbose=2),
-                        f"{method} weights={weights}, axes={axes}, {a!r}, "
-                        f"{b!r}",
                     )
 
             for method in (
                 "mean",
                 "mean_absolute_value",
-                # 'mean_of_upper_decile',
```

Comment:
We probably want to uncomment this line rather than deleting it, as that still exists as a method and therefore should be tested as far as I am aware?

Reply:
Good spot. Line reinstated and uncommented (https://github.com/NCAS-CMS/cf-python/pull/532/commits/6a56fcb62a403d29259080ff46aad4c54d632d3d)